### PR TITLE
LEAF-560: Remediated variables to prevent xss

### PR DESCRIPTION
**Pull request by dastan-rahimi**

### CODE REVIEW

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php [hunk 1]
```diff
@@ -649,6 +649,87 @@ public function testSetCategoryPrivileges_addPriv() : void
...
+      self::$client->postEncodedForm('?a=formEditor/_form_f4687/stapled', array(
+          'stapledCategoryID' => $category['categoryID'],
+      ));
+
+      $category = self::$client->get('?a=formEditor/_form_f4687/stapled')[0];
+      $this->assertNotNull($category);
+      $this->assertEquals('form_f4689', $category['categoryID']);
+
+
+      $delResponse = self::$client->delete('?a=formEditor/_form_f4687/stapled/_form_f4689/');
```
Comment (nathanws) [inline test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: I can't get this test to pass either. Keep getting the error: "Invalid request. Missing Referer"

It seems to be actually deleting the stapled form, but I think the Guzzle method is completing correctly.

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php [hunk 2]
```diff
@@ -338,7 +338,7 @@ public function testSetIndicatorDescription_HTMLinput() : void
         $indicator = self::$client->get('?a=formEditor/indicator/6');
 
         $this->assertNotNull($indicator);
-        $this->assertEquals('&lt;script lang=&#039;javascript&#039;&gt;alert(&#', $indicator['6']['description']);
+        $this->assertEquals('&lt;script lang=&#039;javascript&#039;&gt;alert(&#039;hi&#039;)&lt;/script&gt;<b>stuff</b>', $indicator['6']['description']);
```
[the remark this answers is not in the document]
Comment (dastan-rahimi) [reply test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: this is an existing issue

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php (hunk 1 above)
Comment (dastan-rahimi) [reply test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: you will have to disable "$this->verifyAdminReferrer();" on DELETE in LEAF_Request_Portal/api/controllers/FormEditorController.php to get this test passed. this an existing issue with GUZZLE.

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (nathanws) [reply test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: Can you update the test to reflect the expected value?

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php (hunk 1 above)
Comment (nathanws) [reply test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: Got it.

File: test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php (hunk 2 above)
[the remark this answers is not in the document]
Comment (dastan-rahimi) [reply test/LEAF_Request_Portal_Tests/tests/api/FormEditorControllerTest.php]: done

**Pull request merged**